### PR TITLE
added the app and virtual env

### PR DESCRIPTION
**Pull request by parthasarathi9438**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+asgiref==3.5.2
+Django==4.0.5
+sqlparse==0.4.2
+tzdata==2022.1
+
```
Comment (VishnuK24) [inline requirements.txt]: @parthasarathi9438 Add other requirements packages and dependencies

File: .gitignore [hunk 2]
```diff
@@ -22,6 +22,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+venv/
```
Comment (VishnuK24) [inline .gitignore:27]: @parthasarathi9438 Add migrations folder to .gitignore file and remove all the migration files. We can do the migration manually on the server

File: requirements.txt (hunk 1 above)
Comment (parthasarathi9438) [reply requirements.txt]: resolved

File: .gitignore (hunk 2 above)
Comment (parthasarathi9438) [reply .gitignore:27]: added migrations file in gitignore kindly remove it

**Review by VishnuK24 — APPROVED** [review]

**Pull request merged**